Code Review Assignment_JrSDET.py

```diff
@@ -0,0 +1,118 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+import time
+
+class Testcase101:
+
+    def main(self):
+        driver = webdriver.Firefox(executable_path="C:\\Users\\Johny\\Downloads\\geckodriver-v0.33.0-win64\\geckodriver.exe")
+        driver.get("https://interview.supporthive.com/staff/")
+        driver.implicitly_wait(30)
+        driver.maximize_window()
+        driver.find_element(By.ID, "id_username").send_keys("Agent")
+        driver.find_element(By.ID, "id_password").send_keys("Agent@123")
+        driver.find_element(By.ID, "btn-submit").click()
+        tickets = driver.find_element(By.ID, "ember29")
+        action = ActionChains(driver)
+        action.move_to_element(tickets).perform()
+        statuses = driver.find_element(By.LINK_TEXT, "Statuses")
+        statuses.click()
+        driver.find_element(By.XPATH, "/html/body/div[3]/div/section/section/div/header/button").click()

```

Comment:
Using CSS Selectors instead of XPath would be better for improved readability and performance.